help/destinations/catalog/crm/salesforce.md

```diff
@@ -255,11 +279,13 @@ All [!DNL Adobe Experience Platform] destinations are compliant with data usage
 
 ### Unknown errors encountered while pushing events to the destination {#unknown-errors}
 
-When checking a dataflow run, if you obtain the following error message: `Unknown errors encountered while pushing events to the destination. Please contact the administrator and try again.`
+* When checking a dataflow run, if you obtain the following error message: `Unknown errors encountered while pushing events to the destination. Please contact the administrator and try again.`
```

Comment:
Grammar: When checking a dataflow run, you might obtain the following error message: